plugins/CustomPiwikJs/File.php

```diff
@@ -60,7 +60,7 @@ public function getName()
      */
     public function hasWriteAccess()
     {
-        return $this->hasReadAccess() && is_writable($this->file);
+        return is_writable($this->file);
```

Comment:
FYI: I had it like this on purpose for now and in tests just created the file in advance